tests/configuration.py

```diff
@@ -0,0 +1,105 @@
+import logging
+
+from django.conf import settings
+from django.test import TestCase
+from django.test.utils import override_settings
+
+from django_emarsys.test_helper import MockLoggingHandler
+from django_emarsys import util
+
+
+class ModelTestCase(TestCase):
+    @override_settings()
+    def test_configuration_present(self):
+        logger = logging.getLogger(util.__file__)
+        handler = MockLoggingHandler(level='WARNING')
+        logger.addHandler(handler)
+
+        # remove all EMARSYS_* settings
+        for symbol in dir(settings):
+            if symbol.startswith('EMARSYS_'):
+                delattr(settings, symbol)
+
+        util.validate_configuration()
```

Comment:
Still don't get where it becomes a dict. It'd still be a Settings object? Is there an elegant way to create a Settings object on the fly?
Or do you mean that it wouldn't matter as long as `validate_configuration` only uses `hasattr()` and `getattr()`?